### PR TITLE
Reference topic: list all users

### PR DESCRIPTION
**Pull request by ConJayTech**
Creates a reference topic for GET command to list all users first names as isolated property

### CODE REVIEW

File: docs/api/users-get-users-first-names.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# Get users first names
+
+Returns an array of  [`user`](user) objects that contains only the property specified by the `first_name` parameter, if it exists.
+
+## URL
+
+```shell
+
+{server_url}/users/:first_name
+```
```
Comment (rbwatson) [inline docs/api/users-get-users-first-names.md:14]: This syntax doesn't work for me when I test it.
Are you sure this is supported?

**Review by rbwatson — CHANGES_REQUESTED** [review]
This looks good, but I couldn't get the example to work.
Are you sure this is a supported operation?

Please review and update the topic or explain and include the documentation reference that shows this.

Thanks!

File: docs/api/users-get-users-first-names.md (hunk 1 above)
Comment (ConJayTech) [reply docs/api/users-get-users-first-names.md:14]: I am not sure this is supported. I've spent the last half hour trying to find query parameters that allow a GET request to return just a single property but I can't figure it out. My understanding is that this kind of functionality would have to be built into the API, that it's not an inherent GET capability. So I'm not sure where to go from here. If I signed up for a reference topic that can't actually be described as supported in the API, should I find a different topic?

**Review by rbwatson — CHANGES_REQUESTED** [review]
Rebased branch.
How about change it to "get users by first_name?"
That's supported and just a minor change.

**Comment by ConJayTech** [comment]
Ok, I changed the ref topic as you suggested. Do I need to create a new pull request? Can you see the changes and merge them?

**Review by rbwatson — APPROVED** [review]
Corrected query syntax
Rebased, squashed for merge
LGTM

**Pull request merged**